### PR TITLE
MA_Lesson4_Updates

### PR DESCRIPTION
**Pull request by marlonseb**
Misc updates

### CODE REVIEW

File: Lesson_4_Introduction_to_Pandas_II/Lesson_4.ipynb [hunk 1]
```diff
@@ -6,13 +6,17 @@
    "source": [
```
Comment (ngrajales1) [inline Lesson_4_Introduction_to_Pandas_II/Lesson_4.ipynb:6]: <p>Leave the following in the cell,</p><p> </p><p>titanic_data =&nbsp;</p><p>titanic_data</p><p> </p><p>This way we start the students off</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theLeague-AI/TheLeagueAI_main/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (ngrajales1) [inline Lesson_4_Introduction_to_Pandas_II/Lesson_4.ipynb:6]: <p>is this NB viewer who moved the assignments or are the assignments below the variable? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theLeague-AI/TheLeagueAI_main/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (marlonseb) [reply Lesson_4_Introduction_to_Pandas_II/Lesson_4.ipynb:6]: <p>done</p>

**Review by ngrajales1 — APPROVED** [review]
LGTM

**Pull request merged**